src/test/java/com/mentoring/framework/Config.java

```diff
@@ -34,7 +39,15 @@
             BASE_URL = prop.getProperty("base_url");
         }
         APPLICATION_URL = BASE_URL.concat(":").concat(PORT);
-
+        DATABASE_NAME = prop.getProperty("database_name");
+        if (System.getenv(ENVIRONMENT).equalsIgnoreCase(SKY_XPLORE_LOCALHOST)) {
+            JDBC = "jdbc:mysql://localhost/" + DATABASE_NAME + "?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC";
+        } else {
+            // What should we do on prod? Do we have rights to connect to Prod database?
```

Comment:
You need to create custom exception. See checked custom exception here: https://www.baeldung.com/java-new-custom-exception